go/chat/unfurl/scrape_map.go

```diff
@@ -51,6 +56,9 @@ func (s *Scraper) scrapeMap(ctx context.Context, uri string) (res chat1.UnfurlRa
 		}
 		timeStr = fmt.Sprintf("Posted %s.", time.Now().Format("15:04:05 MST"))
```

Comment:
should drop before release or convert to local time

Reply:
I'll get time eventually, still thinking about it.